bindings/java/src/integration/com/apple/foundationdb/GetClientStatusIntegrationTest.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * GetClientStatusIntegrationTest.java
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2013-2018 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.apple.foundationdb;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Integration tests around Range Queries. This requires a running FDB instance to work properly;
+ * all tests will be skipped if it can't connect to a running instance relatively quickly.
+ */
+class GetClientStatusIntegrationTest {
+	public static final int API_VERSION = 720;
+	private static final FDB fdb = FDB.selectAPIVersion(API_VERSION);
+
+	@Test
+	public void clientStatusIsHealthy() throws Exception {
```

Comment:
There are multiple negative tests of fdb_database_get_client_status at the C API level (fdb_c_client_config_tests.py). For Java and Python, I thought it would be just enough to test if they pass the the result of the C API correctly.